pgpool2-4.6.yaml

```diff
@@ -104,6 +104,7 @@ subpackages:
         - grep
         - gawk
         - gnutar
+        - busybox # needed by entrypoint script
```

Comment:
Is full busybox required or only sh (that's available via dash-binsh)

Reply:
`sed`, `hostname`, etc are required.

Reply:
Dockerfile reference: https://github.com/pgpool/pgpool2_on_k8s/blob/master/pgpool.docker/entrypoint.sh